package.json

```diff
@@ -19,34 +19,34 @@
   "main": "src/index.js",
   "repository": "autonomoussoftware/metronome-sdk-status",
   "scripts": {
-    "check": "dependency-check . && dependency-check . --unused --no-dev",
-    "check:outdated": "npm-check -i husky",
+    "check": "dependency-check ./package.json --unused --no-dev",
```

Comment:
defaults have changed, so in order to check those prod deps not used, this is the proper script